.github_actions.yml

```diff
@@ -0,0 +1 @@
+.github/workflows/main.yml
```

Comment:
What is the purpose of this file? Every time I've added Github Actions via the web interface, it only creates the `.github/workflows/main.yml` file, which seems to be sufficient for CI

Reply:
This is just a symbolic link to have access at the root of the workspace.